WordPress/Classes/ViewRelated/Blog/TimeZoneSelectorViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+
+class TimeZoneSelectorViewController: UITableViewController, ImmuTablePresenter {
+
+    /// users current timezone passed by SiteSettingsVC, if empty means manual offset is to be used
+    @objc var usersCurrentTimeZone: String?
+    /// users manual offset passed by SiteSettingsVC
```

Comment:
I'd avoid mentioning which view controller should set these values – technically, this view controller could be created and presented from anywhere.